guides/doc-Planning_for_Project/master.adoc

```diff
@@ -5,18 +5,13 @@ include::common/header.adoc[]
 
 = {PlanningDocTitle}
 
-// This guide is Katello specific, in particular the diagrams
-ifdef::foreman-el,foreman-deb[]
-include::common/modules/snip_guide-not-ready.adoc[]
-endif::[]
-ifndef::foreman-el,foreman-deb[]
-
 ifdef::satellite[]
 include::common/modules/proc_providing-feedback-on-red-hat-documentation.adoc[leveloffset=+1]
 endif::[]
 
 include::common/modules/con_foreman-overview-and-concepts.adoc[]
 
+ifdef::katello,satellite,orcharhino[]
```

Comment:
Missing `endif` below Content and patch management?

There's not much in the foreman-el build right now :)

Reply:
Yes, my bad. Moved it.